sphinxext/tabs.py

```diff
@@ -275,14 +275,31 @@ def setup(app):
     app.add_directive('tabs-cloud',
         create_tab_directive('cloud', DEPLOYMENTS));
 
-    # Create operating system tab directive
+    # Create client operating system tab directive
     app.add_directive('tabs-platforms',
         create_tab_directive('platforms',
             [('windows', 'Windows'),
              ('macos', 'macOS'),
-             ('linux', 'Linux'),
+             ('debian', 'Ubuntu/Debian'),
+             ('rhel', 'RHEL/CentOS/AMZ'),
+             ('linux', 'Linux')]))
+
+    # Create server operating system tab directive
+    app.add_directive('tabs-server-platforms',
+        create_tab_directive('server-platforms',
+            [
+             ('windows', 'Windows'),
+             ('deb', 'Ubuntu/Debian'),
+             ('rpm', 'RHEL/CentOS/AMZ'),
              ('debian', 'Debian'),
-             ('rhel', 'RHEL')]))
+             ('ubuntu', 'Ubuntu'),
+             ('rhel', 'RHEL/CentOS'),
+             ('suse', 'SUSE'),
+             ('amz', 'Amazon Linux'),
+             ('rhel', 'RHEL/CentOS/AMZ'),
+             ('linux', 'Other Linux')
```

Comment:
Historically the docs team has used the `linux` tabid to refer to Linux in general rather than any specific distro; has this changed?

Reply:
Problem is, with Cloud especially, Linux can mean TAR installs, non-package installers, etc. rather than Linux covering RHEL, Debian, etc.